pkg/lib/feature/verification/store_pq.go

```diff
@@ -35,7 +35,7 @@ func (s *StorePQ) scan(scn db.Scanner) (*Claim, error) {
 		&c.CreatedAt,
 	)
 	if errors.Is(err, sql.ErrNoRows) {
-		return nil, ErrClaimUnverified
+		return nil, nil
```

Comment:
The value of `ErrClaimUnverified` is used in VerificationService. It seems that we have to return ErrClaimUnverified to signify the claim is not verified (or not found in the verified claim table).

Reply:
Sorry, updated it accidentally, removed it:(